bftengine/src/bftengine/SigManager.hpp

```diff
@@ -72,6 +73,13 @@ class SigManager {
   SigManager(SigManager&&) = delete;
   SigManager& operator=(SigManager&&) = delete;
 
+  concord::messages::keys_and_signatures::ClientsPublicKeys clientsPublicKeys_;
+  std::string ClientsPublicKeys() {
```

Comment:
confusing - function name looks like constructor because it starts from a capital letter